### PR TITLE
Gateway

### PR DESCRIPTION
**Pull request by lxning**

### CODE REVIEW

File: gateway/controller.go [hunk 1]
```diff
@@ -165,3 +169,65 @@ func (c *ControllerHTTPClient) GetAssignment(jobNamespace, instance string) (ass
...
+	if err != nil {
+		err = utils.StackError(err, "controller client error fetching schema for table: %s", tableName)
+		return
+	}
+	return
+}
+
+// ExtendEnumCases extends enum cases to given table column
+func (c *ControllerHTTPClient) ExtendEnumCases(tableName, columnName string, enumCases []string) (enumIDs []int, err error) {
+	enumCasesRequest := client.EnumCasesWrapper{
```
Comment (jshencode) [inline gateway/controller.go]: this seems like an unnecessary dependency on client package

Comment (lxning) [reply gateway/controller.go]: removed, done

File: client/connector.go [hunk 2]
```diff
@@ -58,8 +58,8 @@ type UpsertBatchBuilder interface {
 	PrepareUpsertBatch(tableName string, columnNames []string, updateModes []memCom.ColumnUpdateMode, rows []Row) ([]byte, int, error)
 }
 
-// enumCasesWrapper is a response/request body which wraps enum cases
-type enumCasesWrapper struct {
+// EnumCasesWrapper is a response/request body which wraps enum cases
+type EnumCasesWrapper struct {
```
Comment (jshencode) [inline client/connector.go]: maybe not export this struct

Comment (lxning) [reply client/connector.go]: done

**Pull request merged**